integration-cli/requirements_test.go

```diff
@@ -159,8 +159,10 @@ func IsPausable() bool {
 	return true
 }
 
+// These requirement is a noop on non-windows oses. One should
+// expicitly add DaemonIsWindows requirement if needed.
 func IsolationIs(expectedIsolation string) bool {
-	return testEnv.OSType == "windows" && string(testEnv.DaemonInfo.Isolation) == expectedIsolation
+	return testEnv.OSType != "windows" || string(testEnv.DaemonInfo.Isolation) == expectedIsolation
```

Comment:
I'm a bit on the fence on this one, because `IsolationIs("process")` would return true on Linux (but isn't actually the case?)

Should we instead modify the tests to do something like


```go
testRequires(c, testEnv.IsLocalDaemon)
if DaemonIsWindows() {
    testRequires(c, IsolationIsProcess)
}
```

Reply:
Thanks, updated pr according to you comment.